lib/view_model/exchange/exchange_view_model.dart

```diff
@@ -611,4 +631,19 @@ abstract class ExchangeViewModelBase with Store {
         selectedProviders
             .where((provider) => providersForCurrentPair().contains(provider)));
   }
+
+  @action
+  TransactionPriority? setDefaultTransactionPriority() {
```

Comment:
The return type could be void.
Also, I checked the ticket, and yes the priorities are as set to default as the ticket description, but the default transaction priority for haven and monero is different in the wallet code, which one should be the default?

Reference:
`getDefaultTransactionPriority()` in `lib/haven/cw_haven.dart` and in `lib/monero/cw_monero.dart`

Reply:
Better to create new task for change default transaction priorities for haven and monero wallets